### ISSUE TITLE
Opening file:///dev/zero results in memory getting eaten up

### ISSUE THREAD

**Issue by MistressRemilia** [skyjake/lagrange#641]
I accidentally opened file:///dev/zero the other day and found my memory was totally eaten at **very** high speed, causing Lagrange to totally lock and requiring me to `killall -s9 lagrange` in a terminal.  Maybe a few rules for special files should be in place to keep from opening them by accident?

**Comment by skyjake** [comment]
Yes, some of these special files like /dev/zero should be treated as a special case. There could also be some limit on the maximum viewable file size.
